plugins/generator-1.19.2/forge-1.19.2/utils/triggers.java.ftl

```diff
@@ -492,7 +492,7 @@
     <#if procedure?has_content || hasProcedure(procedure)>
 	@Override public boolean makesPiglinsNeutral(ItemStack stack, LivingEntity wearer) {
         <#if hasProcedure(procedure)>
-			Entity entity = stack.getEntityRepresentation();
+			Entity entity = (Entity) wearer;

```

Comment:
Why do you have to cast this `LivingEntity` instance to its superclass? You can just rename the method argument.

Reply:
What method argument?

Reply:
`wearer` to `entity`.